scripts/extraimages/01-debianconsole

```diff
@@ -0,0 +1,17 @@
+FROM debian:jessie
+RUN apt-get update && \
+    apt-get upgrade --no-install-recommends -y && \
+    apt-get install -y --no-install-recommends openssh-server rsync locales vim
+RUN rm -rf /etc/ssh/*key*
+COPY scripts/dockerimages/scripts/entry.sh /usr/sbin/
+COPY scripts/dockerimages/scripts/console.sh /usr/sbin/
+COPY scripts/dockerimages/scripts/update-ssh-keys /usr/bin/
+RUN echo 'RancherOS \\n \l' > /etc/issue
+RUN locale-gen en_US.UTF-8
+RUN addgroup --gid 1100 rancher && \
+    addgroup --gid 1101 docker && \
+    useradd -u 1100 -g rancher -G docker,sudo -m -s /bin/bash rancher && \
+    echo '## allow password less for rancher user' >> /etc/sudoers && \
+    echo 'rancher ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
```

Comment:
@imikushin docker user needs to be added as well, https://github.com/rancherio/os/pull/239/files#diff-e14a520ce1af4fb35a5ef4f0578ebd65R8